CHANGELOG.rst

```diff
@@ -1,3 +1,13 @@
+0.1.7
+-----
+- Adding support to register and run a TANGO device using a file as a database.
+- Initialising the model quantity values with the default values specified in the simulator data description files.
+- Parsing simulator data description files in a pre-determined precendence order.
+- Removing unmaintained python-devicetest dependency.
+- Added a fandango json parser
+    - This parser is used to process the TANGO device data file generated by the [fandango library](https://github.com/tango-controls/fandango).
```

Comment:
This URL link doesn't render nicely on github, if I click "View".  Can it be improved?